zenoh-plugin-mqtt/src/config.rs

```diff
@@ -94,6 +97,10 @@ where
     deserializer.deserialize_option(OptPathVisitor)
 }
 
+fn default_mqtt_tx_channel_size() -> usize {
```

Comment:
addind new functionality should be in another PR
I tinnk the https://github.com/eclipse-zenoh/zenoh-plugin-mqtt/pull/131 should be applied first and this `mqtt_tx_channel_size` update  should be applied after it, better with reference to the corresponding issue